app/src/main/java/com/nitrr/ecell/esummit/ecellapp/misc/hamburgeritem.java

```diff
@@ -0,0 +1,57 @@
+package com.nitrr.ecell.esummit.ecellapp.misc;
+
+import android.annotation.SuppressLint;
+import android.app.Dialog;
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.design.widget.BottomSheetDialogFragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import com.nitrr.ecell.esummit.ecellapp.R;
+
+public class hamburgeritem extends BottomSheetDialogFragment {
```

Comment:
Name of this Java File needs to HamburgerItemBottomSheetDialogFragment or some other relevant name

Reply:
is BottomSheetFragment ok?